.travis.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     - stage: rustfmt
       # Install rustfmt 0.9.0 if it isn't already installed on
       # Travis. This can be slow to install, so raise the Travis timeout.
-      before_script: (which rustfmt && rustfmt --version && [[ "$(rustfmt --version)" =~ "0.9.0" ]]) || travis_wait cargo install --force rustfmt --vers 0.9.0
+      before_script: (which rustfmt && rustfmt --version && [[ "$(rustfmt --version)" =~ "0.2.9" ]]) || travis_wait cargo install --force rustfmt-nightly --vers 0.2.9
```

Comment:
0.9.0 is the latest version.

Reply:
The `rustfmt` crate is no longer updated however, it is only kept around for compatibility with stable Rust. See https://users.rust-lang.org/t/rustfmt-releases/11357 and the linked blog post: New development is happening on a new crate which is recommended.

Of course, this requires a one-time reformat of all the Remacs code and an update to developers' configurations, which might outweigh the benefit of getting the new lints.

Reply:
Oh, I didn't notice `-nightly` in name :)

Reply:
Updated the PR to reflect the actual intent, since I misunderstood what I was doing the first time I submitted it.

Reply:
I've also considered making a PR with the latest rustfmt version, but I thought there's a reason for using the stable version. I think it's a good idea :thumbsup: